### DISCUSSION TITLE
Is jupyter book good for software project documentation?

### DISCUSSION THREAD

**Discussion by cosmicBboy** [executablebooks/meta#1025] [Q&A]
I know it's called "jupyter book", but are there any examples of software libraries/project that use it for their documentation site? Just curious if there are any fundamental things about it that would make that a bad idea.

**Comment by rowanc1** [comment]
JupyterBook is based on top of the Sphinx project, which is used quite extensively for documentation -- especially in the Python ecosystem. The different pieces of JupyterBook can also be used independently directly with Sphinx (e.g. the [book-theme](https://sphinx-book-theme.readthedocs.io/en/latest/reference/api-numpy.html)).

I would say that if your project is in Python, then JupyterBook/Sphinx will be great for the project! If it is based in another language, some of the integrations might not be as easy out of the box, but it would still work well.
